host-controller/src/main/java/org/jboss/as/domain/controller/resources/GlobalHostResourceDefinition.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2015 Red Hat, inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA 02110-1301  USA
+ */
+package org.jboss.as.domain.controller.resources;
+
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.HOST;
+
+import org.jboss.as.controller.PathElement;
+import org.jboss.as.controller.SimpleResourceDefinition;
+import org.jboss.as.controller.operations.global.GlobalInstallationReportHandler;
+import org.jboss.as.controller.registry.ManagementResourceRegistration;
+import org.jboss.as.host.controller.HostModelUtil;
+
+/**
+ * Simple resource for /host=*
+ * @author <a href="mailto:ehugonne@redhat.com">Emmanuel Hugonnet</a> (c) 2015 Red Hat, inc.
+ */
+public class GlobalHostResourceDefinition extends SimpleResourceDefinition {
+    public static final GlobalHostResourceDefinition INSTANCE = new GlobalHostResourceDefinition();
+
+    private GlobalHostResourceDefinition() {
+        super(PathElement.pathElement(HOST), HostModelUtil.getResourceDescriptionResolver());
+    }
+
+    @Override
+    public void registerOperations(ManagementResourceRegistration resourceRegistration) {
+        super.registerOperations(resourceRegistration);
+        resourceRegistration.registerOperationHandler(GlobalInstallationReportHandler.DEFINITION, GlobalInstallationReportHandler.INSTANCE);
+    }
+
+}
```

Comment:
I'm quite certain this isn't something we want to do. Why doesn't the existing stuff for registering host stuff work?